.github/workflows/release.yml

```
@@ -32,18 +32,49 @@ jobs:
+            echo "=============================================="
+            df -h / | tail -1
+
+            bash ./scripts/release-region.sh "$region"
+
+            echo "Disk after $region:"
+            df -h / | tail -1
+          done
+
+      # 5. Generate checksums for all archives
+      - name: Generate checksums
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2035:info:2:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)